modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -260,7 +264,7 @@ const Quirks = (editor: Editor): Quirks => {
     editor.on('mousedown', (e) => {
       Optionals.lift2(Optional.from(e.clientX), Optional.from(e.clientY), (clientX, clientY) => {
         const caretPos = editor.getDoc().caretPositionFromPoint(clientX, clientY);
-        const img = caretPos?.offsetNode.childNodes[caretPos.offset - (caretPos.offset > 0 ? 1 : 0)] || caretPos?.offsetNode;
+        const img = caretPos?.offsetNode?.childNodes[caretPos.offset - (caretPos.offset > 0 ? 1 : 0)] || caretPos?.offsetNode;
 
         if (img && isEditableImage(img)) {
```

Comment:
I'd suggest we make a small change here:

```suggestion
        if (Type.isNonNullable(img) && isEditableImage(img)) {
```

Reply:
refactored using `Type.isNonNullable` because the `??` operator doesn't work in this case since it returns the right sight if the left side is `null`/`undefined`, so in this case we would have `isEditableImage` called with `null`/`undefined`

Reply:
Right, I was thinking about something else. Apologies for the weird request.